capstone/src/main/java/com/google/sps/servlets/BlackOwnedRestaurantsDataServlet.java

```diff
@@ -60,40 +56,51 @@ public void init() {
   }
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-  }
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {}
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    clearDatastore();
     for (String restaurantName : blackOwnedRestaurants) {
       PlaceDetails place = details.request(restaurantName);
 
-      detailedPlaces.add(place); 
+      detailedPlaces.add(place);
 
       String reviews = "";
       PlaceDetails.Review[] reviewsList = place.reviews;
       for (PlaceDetails.Review review : reviewsList) {
         reviews += review.text + " ";
-      } 
-      
-    //   Set<String> tags = details.getTags(reviews);
-    //   String tagsString = tags.toString();
+      }
+
+      //   Set<String> tags = details.getTags(reviews);
+      //   String tagsString = tags.toString();
```

Comment:
We commented out tags and tagsString out as we are exploring a more manual approach to matching tags vs. NLP API which is inconsistent, buggy, and slow. We don't delete just so it is easier to transition back to NLP API if we find that manual approach is slower.